spotify/main.py

```diff
@@ -158,6 +163,11 @@ def api_healthcheck(open_podcast_client):
 
 
 def main():
+
+    # Convert start and end date to datetime objects
+    start_datetime = dt.datetime.strptime(START_DATE, "%Y-%m-%d")
```

Comment:
we should check the format here and raise an error in case the date is not valid or end is smaller than start